src/registrar/migrations/0038_create_groups_v02.py

```diff
@@ -0,0 +1,36 @@
+# This migration creates the create_full_access_group and create_cisa_analyst_group groups
+# It is dependent on 0035 (which populates ContentType and Permissions)
+# If permissions on the groups need changing, edit CISA_ANALYST_GROUP_PERMISSIONS
+# in the user_group model then:
+# [NOT RECOMMENDED]
+# step 1: docker-compose exec app ./manage.py migrate --fake registrar 0035_contenttypes_permissions
+# step 2: docker-compose exec app ./manage.py migrate registrar 0036_create_groups
+# step 3: fake run the latest migration in the migrations list
+# [RECOMMENDED]
+# Alternatively:
+# Only step: duplicate the migration that loads data and run: docker-compose exec app ./manage.py migrate
```

Comment:
```suggestion
# step 1: duplicate the migration that loads data
# step 2: docker-compose exec app ./manage.py migrate
```
Nitpicky but just a small suggestion for consistency. Totally optional

Reply:
Thank you for the suggestion! Just added your suggestions to improve consistency :~)